rumqttd/src/main.rs

```diff
@@ -17,7 +17,7 @@ struct CommandLine {
         default = "PathBuf::from(\"config/rumqttd.conf\")"
     )]
     config: PathBuf,
-    #[cfg(feature = "prof")]
+    #[cfg(all(feature = "prof", not(target_env = "msvc")))]
```

Comment:
This ensures that pprof is not compiled even if unknowingly the feature "prof" is enabled for msvc targets